src/Codeception/Module/REST.php

```diff
@@ -575,11 +575,8 @@ public function sendUNLINK($url, array $linkEntries)
     protected function execute($method, $url, $parameters = [], $files = [])
     {
         // allow full url to be requested
-        if (strpos($url, '://') === false) {
-            $url = $this->config['url'] . $url;
-            if ($this->config['url'] && strpos($url, '://') === false && strpos($url, '/') !== 0 && $this->config['url'][0] !== '/') {
-                $url = '/' . $url;
-            }
+        if (strpos($url, '://') === false && $this->config['url']) {
+            $url = rtrim($this->config['url'], '/') . ($url && $this->config['url'] ? '/' : '') . ltrim($url, '/');
```

Comment:
There is no need to check `$this->config['url']` in ternary operator, it was checked in a previous line.
Another edge case to test is when there is a trailing slash in `config['url']` - `array('http://v1/', '', 'http://v1/'),`
removing the slash could be a wrong thing to do, right?

Reply:
How about handling empty `$url` as a special case?
```php
if ($url === '') {
  $url = $this->config['url'];
} else {
  $url = rtrim($this->config['url'], '/') . '/' . ltrim($url, '/')
}
```

Reply:
sure, done.